CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/player/MCFoodStats.java

```diff
@@ -60,6 +60,6 @@ public float getSaturationLevel() {
     
     @Override
     public void setSaturationLevel(float saturationLevel) {
-        foodStats.setFoodSaturationLevel(saturationLevel);
+        foodStats.addExhaustion(saturationLevel - getSaturationLevel());

```

Comment:
Why this change? Didn't it do what it meant to do before?

Reply:
the `setFoodSaturationLevel` method is `@SideOnly(CLIENT)` so calling it on server side would get a crash.